cranelift/filetests/src/test_zkasm.rs

```diff
@@ -80,30 +80,25 @@ mod tests {
     /// inside given function.
     /// Label name must begin from label_.
     fn optimize_labels(code: &[&str], func_index: usize) -> Vec<String> {
-        let mut label_definition: HashMap<usize, usize> = HashMap::new();
-        let mut label_uses: HashMap<usize, Vec<usize>> = HashMap::new();
+        let mut label_definition: HashMap<String, usize> = HashMap::new();
+        let mut label_uses: HashMap<String, Vec<usize>> = HashMap::new();
```

Comment:
Now labels can have same number but different name, like ``label_5`` and ``label_jmpnz_5``, so now we need use whole label name here, not only index

Reply:
Would it be possible to split this change into a separate PR, given that it is not related to i32 overflows.
This would:
- Allow this change to submitted faster by not being blocked on review of the rest of the code
- Make both changes smaller and easier to review

Reply:
Reverted